docs/glossary.md

```diff
@@ -0,0 +1,5 @@
+# Glossary
+
+* **Native**: By something native, we mention something living on the starknet network (eg: native ETH = StarknetETH)
+* **KEVM**: By something KEVM, we mention something living on the kakarot contract but with additional admin properties managed by the kakarot contract (eg: KEVM WETH)
```

Comment:
same here, nothing lives on the Kakarot contract per se, Kakarot only registers the address at which the account registry lives (see below to understand what is the address registry or account registry)